### PR TITLE
35 create product details page

### PR DESCRIPTION
**Pull request by dmitry-honcharuk**

### CODE REVIEW

File: client/modules/admin/components/ProductDetails/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    }),
+  ).isRequired,
+  isEdit: PropTypes.bool.isRequired,
+  handleSubmit: PropTypes.func.isRequired,
+};
+
+const ProductDetails = ({ productFields, isEdit, handleSubmit }) => {
+  const onSubmit = useCallback(
+    handleSubmit(() => {
+      console.info('SUBMIT');
```
Comment (Danteisabck) [inline client/modules/admin/components/ProductDetails/index.js]: do we need this?

File: client/modules/admin/components/ProductDetails/EditButton.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+
+const EditButton = () => {
+  return (
+    <Button
+      variant="contained"
+      color="primary"
+      onClick={() => console.info('CLICK')}
```
Comment (Danteisabck) [inline client/modules/admin/components/ProductDetails/EditButton.js]: do we need this?

Comment (dmitry-honcharuk) [reply client/modules/admin/components/ProductDetails/EditButton.js]: Just a placeholder, following PR adds functionality here

File: client/modules/admin/components/ProductDetails/index.js (hunk 1 above)
Comment (dmitry-honcharuk) [reply client/modules/admin/components/ProductDetails/index.js]: Just a placeholder, following PR adds functionality here

**Pull request merged**